config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   root "users#feed"
 
+
   devise_for :users
   
   resources :comments
```

Comment:
Users are able to go to /comments, /follow_requests, /likes and edit/delete them. You should consider adjusting the routes to be:

`resources :comments, except: [:index, :show]`

You'll want to do this for comments, likes, and follow_requests.